tests/test_connection_pool.py

```diff
@@ -17,6 +17,15 @@ def __init__(self, **kwargs):
         self.kwargs = kwargs
         self.pid = os.getpid()
 
+    def connect(self):
```

Comment:
This block seems funny, are these just place holders or can they be removed?

Reply:
Don't this this will cause and issue so non-blocking comment.

Reply:
These were added to prevent breaking in dummy-faked tests. I'm guessing that the newer version of `StrictRedis` increased its interactions with the `Connection` type to include these, so the tests blew up without these as fake surface.